pkgs/development/python-modules/pyqt/sip.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "PyQt5_sip";
```

```suggestion
  pname = "pyqt5-sip";
```
normalised it should look like this.

---

> normalised it should look like this.

Done.